sass/_FacetSearch.scss

```diff
@@ -1,7 +1,11 @@
 @import 'Variables';
 @import 'FacetValues';
+@import 'FacetVariables';
+
 .coveo-facet-search-results {
   position: absolute;
+  left: $facet-value-padding;
+  right: $facet-value-padding;
```

Comment:
These two attributes are enough to replace `PopupUtils` in `FacetSearch`, which affects `Facet` and `CategoryFacet`.